packages/bulkapi/src/controllers/BulkApiWorker.ts

```diff
@@ -17,27 +18,39 @@ export class BulkApiWorker {
         return await this.context.queries.getPartitions()
     }
 
+    /**
+     * @param chunk
```

Comment:
I think we wither describe the parameter or we can remove the content

Reply:
Added description